packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -82,7 +84,8 @@ export class ColorPickerBase extends BaseComponent<IColorPickerProps, IColorPick
                 <td>
                   <TextField
                     className={classNames.input}
-                    value={color.hex}
+                    value={color.hex || ''}
+                    onChange={noOp /* included to prevent warnings */}
```

Comment:
Hmm, I'm not sure if I like this. I would consider `ColorPicker`'s use of `TextField` a legitimate use of a controlled component, it just so happens to read the value in `onBlur` instead of `onChange`. If we have to do this here, our consumers will have to do it too, and it feels a bit frictiony. Are we sure we want to *require* `onChange` to exist if value is, even when other callbacks are defined?

Reply:
Actually based on our review `ColorPicker` no longer works and needs to be changed to use `onChange`

Reply:
I reverted this and made the color picker use the old non-strict mode for now.